### PR TITLE
Add argument to lint for passing whitelist path

### PR DESCRIPTION
**Pull request by jeffcarp**
We're looking to run the linter as a presubmit script in Chromium but since the lint module will live in a separate location from the whitelist, it needs to be passed in as a parameter.

Chromium bug: https://crbug.com/686927

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/wpt-tools/202)

### CODE REVIEW

**Review by gsnedders — APPROVED** [review]
Though @jgraham is in the middle of merging this into web-platform-tests, so we probably need to wait till that for this to land.

**Comment by jeffcarp** [comment]
Got it, I can make a new PR in WPT once it merges.

**Comment by foolip** [comment]
@jgraham, is unirepoing imminent?

**Comment by jgraham** [comment]
Yeah, should get done tomorrow.

**Comment by foolip** [comment]
Very exciting :)

**Comment by jeffcarp** [comment]
Superceded by https://github.com/w3c/web-platform-tests/pull/6207

**Comment by jeffcarp** [comment]
For running WPT lint in Chromium I ended up adding a `--repo-root` flag:
https://chromium-review.googlesource.com/c/526398/15/third_party/WebKit/Tools/Scripts/webkitpy/thirdparty/wpt/wpt/tools/lint/lint.py

However I'm not sure if it's worth PRing upstream to WPT since keeping the lint script separate from the WPT repo is potentially a niche use case encountered only by Chromium. If anyone thinks it'd be more widely applicable, I can make a new PR with those changes to WPT.

**Pull request closed without merging**
